header_for_5_lab.h

```diff
@@ -10,15 +10,15 @@ class Flower {
     string color;
```

Comment:
in convention said: Data members of classes, both static and non-static, are named like ordinary nonmember variables, but with a trailing underscore.